pimd/pim_msdp.c

```diff
@@ -1353,66 +1315,7 @@ static void pim_msdp_mg_mbr_free(struct pim_msdp_mg_mbr *mbr)
 	XFREE(MTYPE_PIM_MSDP_MG_MBR, mbr);
 }
 
-static struct pim_msdp_mg_mbr *pim_msdp_mg_mbr_find(struct pim_instance *pim,
-						    struct in_addr mbr_ip)
-{
-	struct pim_msdp_mg_mbr *mbr;
-	struct listnode *mbr_node;
-
-	if (!pim->msdp.mg) {
-		return NULL;
-	}
-	/* we can move this to a hash but considering that number of peers in
-	 * a mesh-group that seems like bit of an overkill */
-	for (ALL_LIST_ELEMENTS_RO(pim->msdp.mg->mbr_list, mbr_node, mbr)) {
-		if (mbr->mbr_ip.s_addr == mbr_ip.s_addr) {
-			return mbr;
-		}
-	}
-	return mbr;
-}
-
-enum pim_msdp_err pim_msdp_mg_mbr_add(struct pim_instance *pim,
```

Comment:
It was hard for me to understand that pim_msdp_mg_peer_xxx and pim_msdp_peer_xxx are actually two different things because of the lack of uniform function names (related to the reorg in this change). To make the code readable I will suggest the following changes -
1. Rename pim_msdp_mg_add_peer pim_msdp_mg_mbr_add 
2. Rename pim_msdp_mg_mbr_do_del to pim_msdp_mg_mbr_del and make it pim_msdp_mg_mgr_add's counterpart

This way the pim_msdp_mg_mbr functions are for managing the mbr_list per mesh-groupand pim_msdp_peer functions are for managing the peer list.